README.md

```diff
@@ -272,6 +273,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 [meeting-recruitment-app#js]:samples/meeting-recruitment-app/nodejs
 [qrappinstallation#csharp]:samples/app-installation-using-qr-code/csharp
 [qrappinstallation#nodejs]:samples/app-installation-using-qr-code/nodejs
+[qrjointeam#csharp]:samples/join-a-team-using-qr-code/csharp
```

Comment:
Sample name should be start with bot- like "bot-join-a-team-using-qr-code"

Reply:
Since this is a scenario specific sample and not bot so did not add bot naming convention for this